tests/utils/index.ts

```diff
@@ -1,4 +1,6 @@
+import fs from "fs";
```

Comment:
An FYI comment re `utils` as this dir name: https://go.dev/blog/package-names#bad-package-names (I think this structure is fine as is right now).

(Personally, I think a kitchen-drawer library is always necessary, since it removes some amount of mental decision making around "where should this go" for stuff that has no clear home; but I do agree that we should always be looking for a path to move things out of the kitchen sink.)

Reply:
Had the same train of thought. I think right now it's fine, but I intend to extract things out as it expands and begins to take on the flavor of any particular category.